pkg/config/config.go

```diff
@@ -77,8 +77,9 @@ type proxyLocalConfig struct {
 // LocalConfigInfo wraps the local config and provides helpers to
 // serialize it.
 type LocalConfigInfo struct {
-	Filename    string `yaml:"FileName,omitempty"`
-	LocalConfig `yaml:",omitempty"`
+	Filename         string `yaml:"FileName,omitempty"`
+	LocalConfig      `yaml:",omitempty"`
+	ConfigFileExists bool
```

Comment:
Needs to be an un-exported field exposed if required only be a getter and no setter needs to be there
as this ideally needs to be set only by New... atleast for now

Reply:
Makes sense, updating